### ISSUE TITLE
missing public key

### ISSUE THREAD

**Issue by Rageoholic** [Malabarba/spinner.el#19]
When I go to install a mode with this as a requirement I get an error that this package's public key is missing. Do you need to make a new point release to fix that

**Comment by john2x** [comment]
Running into this as well. Can't install  `company-lsp` because it (or one of its dependencies) depend on spinner.el

**Comment by a-hausmann** [comment]
Indeed, spinner has disappeared from MELPA, and now paradox getting an error about missing spinner: "Eager macro-expansion failure: ... elpa/spinner-1.7.3/spinner.elc failed to provide feature ‘spinner’".  Copying zipped package to elpa directory yields worse message, "Unable to activate package ‘paradox’. Required package ‘spinner-1.7.3’ is unavailable".

**Comment by Malabarba** [comment]
Hey all. Sorry for the radio silence here. 

@Rageoholic and @john2x 
Unfortunately, this thread doesn't make a lot of sense to me as is. "Missing public key" is not something I've ever seen in the Elpa ecossystem. If you're still having this problem, please paste the full error logs.

Finally, the error from @a-hausmann seems unrelated. So please open a new issue if you sill have it. Although, honestly, it looks like some byte-compilation issue unrelated to the package and it might be that you just need to remove and reinstall some (maybe many) packages.

Also note that spinner.el was never on Melpa (and has no need to be). So it most certainly hasn't "disappeared from Melpa".

**Comment by moniquelive** [comment]
@Malabarba I have the following error trying to install spinner today. Maybe it's the same error reported here:

```shell
Failed to verify signature spinner-1.7.4.tar.sig:
Signature made by expired key 066DAFCB81E42C40 GNU ELPA Signing Agent (2019) <elpasign@elpa.gnu.org>
Command output:
gpg: Signature made Fri Jul  2 06:10:02 2021 -03
gpg:                using RSA key C433554766D3DDC64221BFAA066DAFCB81E42C40
gpg: Good signature from "GNU ELPA Signing Agent (2019) <elpasign@elpa.gnu.org>" [expired]
gpg: Note: This key has expired!
Primary key fingerprint: C433 5547 66D3 DDC6 4221  BFAA 066D AFCB 81E4 2C40
```

The current elpa key is as of today:

```shell
pub   ed25519 2022-12-28 [C] [expires: 2032-12-25]
      AC49B8A5FDED6931F40EE78BF993C03786DE7ECA
uid           [ unknown] GNU ELPA Signing Agent (2023) <elpasign@elpa.gnu.org>
sub   ed25519 2022-12-28 [S] [expires: 2027-12-27]
```

TIA.

**Comment by Malabarba** [comment]
Hi there.
The whole packaging and signing stuff is handled by the GNU Elpa repository. 
This might have been just some oversight, where someone forgot to renew the key. :-)

Are you still having the issue? If so, you'll certainly find some help in the GNU Elpa bug tracker
